javaparser-core-metamodel-generator/src/main/java/com/github/javaparser/generator/metamodel/MetaModelGenerator.java

```diff
@@ -196,7 +199,9 @@ public static void main(String[] args) throws Exception {
                 .setLanguageLevel(ParserConfiguration.LanguageLevel.RAW)
                 .setStoreTokens(false);
         final SourceRoot sourceRoot = new SourceRoot(root, parserConfiguration);
-        sourceRoot.setPrinter(new PrettyPrinter(new PrettyPrinterConfiguration().setEndOfLineCharacter("\n"))::print);
+        ConfigurablePrinter config = new PrinterConfiguration().addOption(ConfigOption.END_OF_LINE_CHARACTER.value("\n"));
+        Printable printer = new PrettyPrintable(config);
```

Comment:
A printer is a printable? I don't want to print the printer?

Reply:
I am open to all naming proposals. If you have ideas don't hesitate.

Reply:
Perhaps a Printer interface is more intuitive. PrettyPrinter is a Printer, it make a sense.

Reply:
Renamed Printer